### PR TITLE
CA-241431: Using standard .net WebClient to download updates.xml

### PR DESCRIPTION
**Pull request by GaborApatiNagy**
Switched to use WebClient when downloading updates.xml
Fixed a bug where proxy settings were ignored when downloading updates

### CODE REVIEW

**Comment by stephen-turner** [comment]
Are there any other places where we use HTTPHelper.GET that will cause problems?
Conversely, is there anything our HTTPHelper.GET does that WebClient doesn't do?

File: XenModel/Actions/Updates/DownloadAndUnzipXenServerPatchAction.cs [hunk 1]
```diff
@@ -87,6 +87,9 @@ private void DownloadFile()
                 {
                     try
                     {
+                        var proxy = XenAdminConfigManager.Provider.GetProxyFromSettings(Connection);
```
Comment (MihaelaStoica) [inline XenModel/Actions/Updates/DownloadAndUnzipXenServerPatchAction.cs]: I believe this has been fixed by @Frezzle on the CAR-2214 branch

**Comment by GaborApatiNagy** [comment]
re @stephen-turner 's comment: There will be no other places after this PR, because we'll use HTTPHelper.GET only at places where we communicate with the API. 

Following this PR
* for XS hosts (to the API) we'll use the SDK's implementation (this is already the case), 
* for the web, we'll use only the WebClient class (this is the case after this PR).

File: XenModel/Actions/Updates/DownloadAndUnzipXenServerPatchAction.cs (hunk 1 above)
Comment (GaborApatiNagy) [inline XenModel/Actions/Updates/DownloadAndUnzipXenServerPatchAction.cs]: As we agreed, we'll solve this if it becomes a merge conflict at all.

**Comment by MihaelaStoica** [comment]
Looks good. Merging.

**Pull request merged**